src/Spago/Config.purs

```diff
@@ -334,18 +439,26 @@ readWorkspace { maybeSelectedPackage, pureBuild, migrateConfig } = do
             logDebug "Lockfile is up to date, using it"
             pure (Right contents)
 
-  -- Read in the package database
+loadPackageSet
+  :: ∀ a
+   . { workspaceConfig :: Core.WorkspaceConfig
+     , loadedPackages :: Map PackageName WorkspacePackage
+     , rootPath :: RootPath
+     , lockfile :: Either String Lockfile
+     }
+  -> Spago (Registry.RegistryEnv a) { packageSet :: PackageSet, compiler :: Range }
+loadPackageSet { lockfile, workspaceConfig, loadedPackages, rootPath } = do
```

Comment:
```suggestion
loadPackageSet { lockfile: eitherLockfile, workspaceConfig, loadedPackages, rootPath } = do
```

Reply:
I will change this since you asked, but this is actually how it's in the final result: https://github.com/fsoikin/spago/blob/3c2b1ce92a5d9d1edbd236181eb821cabafd9c6a/src/Spago/Config.purs#L136-L139

Reply:
Renamed.